docs/apps/lsdyna.md

```diff
@@ -0,0 +1,38 @@
+## LS-DYNA
+
+LS-DY­NA is a gen­er­al-pur­pose fi­nite el­e­ment pro­gram ca­pa­ble of sim­u­lat­ing com­plex re­al world prob­lems. It is used by the au­to­mo­bile, aero­space, con­struc­tion, mil­i­tary, man­u­fac­tur­ing, and bio­engi­neer­ing in­dus­tries. LS-DY­NA is op­ti­mized for shared and dis­trib­uted mem­o­ry Unix, Lin­ux, and Win­dows based, plat­forms.
+
+## License
+
+LS-DYNA is proprietary software. The licenses available at CSC are only for academic use.  
+
+## Available
+
+CSC's LS-DYNA licenses are available on CSC's computing platforms [Taito and Puhti](https://research.csc.fi/csc-s-servers).  Latest products will be available on the servers, and earlier versions installation is also possible.  All installed versions are maintained on the servers.
+
+Notice that Taito server will be decommisioned in the end of December 2019.
+
+## Usage
+
+After login on the server, give command
+
+    module available
+
+and check lines *lsdyna/(version number)*. For example to load LS-DYNA version 11.1.0, give command
+
+    module load lsdyna/11.1.0
+
+There is available an example of **batch job file** on Puhti server:
+
+    /appl/soft/eng/lsdyna/example_batch_job_files/parjob_puhti_lsdyna
```

Comment:
This example is not readable
`-rwx------. 1 eskoj p_installation`

Reply:
Corrected.